packages/mui-base/src/utils/reactVersion.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+let version = 17;
```

Comment:
React has a `version` export that's supported across all the majors

Reply:
TIL, thanks!